distro/openshift-template/mt/apicurio-registry-mt-template.yaml

```diff
@@ -784,4 +801,4 @@ parameters:
 
 - name: CORS_ALLOWED_ORIGINS
```

Comment:
Consider adding the description for the new environment variables here, like the already existing ones.